assets/scss/components/_navigation.scss

```diff
@@ -90,6 +90,17 @@
     cursor: pointer;
 }
 
+.c-navigation__icon {
+    max-width: rem-calc(24px);
+    max-height: rem-calc(24px);
+    fill: transparent;
+
+    &.c-menu__close {
```

Comment:
What's the reason for mixing two different components in one element? Can't we have `c-navigation__icon--close` ?